### PR TITLE
Fix/15517287 select options content

### PR DESCRIPTION
**Pull request by KatarzynaStefanowiczPagepro**
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

### CODE REVIEW

**Comment by KatarzynaStefanowiczPagepro** [comment]
https://tw.auroracreation.com/app/tasks/15517287

image

Added initial extension data, put proper data to accordion

File: src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx [hunk 1]
```diff
@@ -200,13 +206,26 @@ export const useLazyConsensusPage = (
             complementaryLabel,
             maxValue,
             validation,
+            accordionItemDescription,
           }) => {
+            const subTitleText = description.defaultMessage
+              ? description.defaultMessage.split('\n')[0]
+              : description;
+
+            let descriptionText: any;
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx:215]: please add correct type

**Review by CzarekDryl — APPROVED** [review]

File: src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx (hunk 1 above)
Comment (krawczykkatarzyna) [reply src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx:215]: maybe true to use useState and on useEffect change the descriptionText?

**Review by krawczykkatarzyna — APPROVED** [review]

File: src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx (hunk 1 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx]: Removed

**Pull request merged**